plutus-ir/src/Language/PlutusIR/Optimizer/DeadCode.hs

```diff
@@ -90,5 +91,6 @@ processBinding
     -> m (Binding tyname name a)
 processBinding = \case
     TermBind x d rhs -> TermBind x d <$> processTerm rhs
+-- MERGE CONFLICT
```

Comment:
There was a lot of stuff here in your `imp/pir-ifix` branch. I guess it's outdated?